Specs/Core/AxisAlignedBoundingBoxSpec.js

```diff
@@ -161,4 +161,30 @@ defineSuite([
             AxisAlignedBoundingBox.intersectPlane(box, undefined);
         }).toThrowDeveloperError();
     });
+
+    var scratchCartesian3 = new Cartesian3();
+    var secondScratchCartesian3 = new Cartesian3();
+    it('projectedPoint given input value outside of box', function() {
+        var upperBound = Cartesian3.add(Cartesian3.add(Cartesian3.UNIT_X, Cartesian3.UNIT_Y, scratchCartesian3),
+            Cartesian3.UNIT_Z, scratchCartesian3);
+        var lowerBound = Cartesian3.subtract(Cartesian3.ZERO, upperBound, secondScratchCartesian3);
+        var box = new AxisAlignedBoundingBox(lowerBound, upperBound);
+
+        var testLocation = new Cartesian3(3, 3, 3);
+        var result = box.clampToBounds(testLocation, scratchCartesian3);
+
+        expect(result).not.toEqual(testLocation);
```

Comment:
A more precise test would be good here.  Checking that it doesn't equal the original value doesn't mean that it returned the correct value.

Reply:
done.